### PR TITLE
Ethdenver2019 PR Part 3

### PR DESCRIPTION
**Pull request by kuhnchris**
### See other PRs

### CODE REVIEW

**Comment by kuhnchris** [comment]
Screenshot:
image

Want me to adapt the text @owocki ?

File: app/event_ethdenver2019/views.py [hunk 1]
```diff
@@ -80,7 +80,12 @@ def ethdenver2019(request):
         "page2": request.GET and recv_addr != 'invalid',
         "page3": all_kudos_collected
     }
+
     page_ctx["page1"] = not page_ctx['page2']
+    if page_ctx["page1"] == True:
```
Comment (owocki) [inline app/event_ethdenver2019/views.py]: shouldnt this be triggered by something dynamic, like the system knowing that hte user collected all kduos?

**Comment by owocki** [comment]
what do i have to do to tell the system what kudos count as ethdenver kudos? i guess i just enter new rows into ethdenver/models.py:Event_ETHDenver2019_Customizing_Kudos ?

**Comment by owocki** [comment]
prod kudos loaded up!! http://bits.owocki.com/b0fc849dcec4/Screen%20Shot%202019-02-11%20at%204.13.07%20PM.png

File: app/event_ethdenver2019/views.py (hunk 1 above)
Comment (kuhnchris) [reply app/event_ethdenver2019/views.py]: This is just to avoid that the page3 (== the 'hooray') is displayed on the "Get started" page.

Comment (owocki) [reply app/event_ethdenver2019/views.py]: @kuhnchris does the system recognize when all of the kudos have been captured, and show `page3` ? if so, where can i see that code?

Comment (owocki) [reply app/event_ethdenver2019/views.py]: @kuhnchris does the system recognize when all of the kudos have been captured, and show `page3` ? if so, where can i see that code?

Comment (kuhnchris) [reply app/event_ethdenver2019/views.py]: Assuming all Kudos are collected...
https://github.com/gitcoinco/web/blob/c2e00abbb11afa0876918902e0ce9ba638c4bdb8/app/event_ethdenver2019/views.py#L56
If not then revert:
https://github.com/gitcoinco/web/blob/c2e00abbb11afa0876918902e0ce9ba638c4bdb8/app/event_ethdenver2019/views.py#L70
And then page3 (which isn't a page anymore but the 'popover') is set to that value:
https://github.com/gitcoinco/web/blob/c2e00abbb11afa0876918902e0ce9ba638c4bdb8/app/event_ethdenver2019/views.py#L78-L82

**Comment by kuhnchris** [comment]
> what do i have to do to tell the system what kudos count as ethdenver kudos? i guess i just enter new rows into ethdenver/models.py:Event_ETHDenver2019_Customizing_Kudos ?

yes, and yes. The only obsolete thing is the "final" thing, that would have been used for the distribution.


> prod kudos loaded up!! http://bits.owocki.com/b0fc849dcec4/Screen%20Shot%202019-02-11%20at%204.13.07%20PM.png

Looks awesome! Checked it on live, looks incredible! Did you test it with a eth address giving them all the required kudos and see if page3 shows up?

**Pull request merged**